libs/remix-ui/renderer/src/lib/renderer.tsx

```diff
@@ -101,7 +101,20 @@ export const Renderer = ({message, opt = {}, plugin}: RendererProps) => {
             <span className="ml-3 pt-1 py-1" >
               <CopyToClipboard content={messageText} className={` p-0 m-0 far fa-copy ${classList}`} direction={'top'} />
             </span>
-            <span className="border border-success text-success btn-sm" onClick={() => { askGtp() }}>ASK GPT</span>
+            <span
+              className="position-relative text-ai text-sm pl-0 pr-2"
+              style={{fontSize: "x-small", alignSelf: "end"}}
+            >
+              AI
```

Comment:
![remix](https://github.com/ethereum/remix-project/assets/6940742/e2d533fb-abfd-44d0-9d76-c024722a21ff)
Are you sure about showing this like that. This looks pretty weird @LianaHus

Reply:
@yann300 

> getting this when trying the code suggestion feature

this error originates from the code completion server. This is returned when the server is down 

> when loading Remix, it shows directly Solidity copilot deactivated! , is this wanted?

Yes, this is desired. A log is also prompted whenever the user turns ON the copilot 

> When hovering the action which activate the AI features, it says that the model is downloaded. There's no way to know that the model is actually downloaded. I actually get "Unexpected token '<' ,... " (see screenshot above). Does this error means taht the model wasn't downloaded correctly?

Do you have an image with the "model downloaded"? Anything related to the inbrowser model should be discarded by now


> How should we tell users which features come with enabling the AI? (msg in the tooltip, link to documentation, .. ?) 

Maybe the link to the documentation will be more appropriate. Or get them to know it by discovering it intuitively